static-site/src/pages/groups.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import ScrollableAnchor from "react-scrollable-anchor";
+import { MdPerson } from "react-icons/md";
+import { SmallSpacer, HugeSpacer, FlexCenter } from "../layouts/generalComponents";
+import * as splashStyles from "../components/splash/styles";
+import { fetchAndParseJSON } from "../util/datasetsHelpers";
+import DatasetSelect from "../components/Datasets/dataset-select";
+import { GroupCards } from "../components/splash/userGroups";
+import GenericPage from "../layouts/generic-page";
+import UserDataWrapper from "../layouts/userDataWrapper";
+
+const nextstrainLogoPNG = require("../../static/logos/favicon.png");
+
+const title = "Scalable Sharing with Nextstrain Groups";
+const abstract = (<>
+  We want to enable research labs, public health entities
+  and others to share their datasets and narratives through Nextstrain with
+  complete control of their data and audience. Nextstrain Groups is more scalable
+  than community builds in both data storage and viewing permissions. Datasets in
+  a public group are accessible to the general public via nextstrain.org, while
+  private group data are only visible to logged in users with permission to see
+  the data. A single entity can manage both a public and a private group in order
+  to share data with different audiences.
+  <br/>
+  <br/>
+  For more details about Nextstrain Groups
+  , <a href="https://docs.nextstrain.org/en/latest/guides/share/nextstrain-groups.html">check out our documentation</a>.
+  <br/>
+  <br/>
+  Nextstrain Groups is still in the early stages and require a Nextstrain team
+  member to set up and add users.
+  Please <a href="mailto:hello@nextstrain.org">contact us at hello@nextstrain.org </a> and we’d be happy to set up a group for you.
+</>);
+const tableColumns = [
+  {
+    name: "Dataset",
+    value: (dataset) => dataset.filename.replace(/_/g, ' / ').replace('.json', ''),
+    url: (dataset) => dataset.url
+  },
+  {
+    name: "Group Name",
+    value: (dataset) => dataset.contributor,
```

Comment:
Please add comment here that the key `contributor` is specific to the JSON being fetched, which may change in a subsequent PR, and actually corresponds to the group name in this case.